decidim-core/db/migrate/20161108093802_create_decidim_static_pages.rb

```diff
@@ -0,0 +1,11 @@
+class CreateDecidimStaticPages < ActiveRecord::Migration[5.0]
+  def change
+    create_table :decidim_static_pages do |t|
+      t.hstore :title, null: false
```

Comment:
I've created an issue to fix this: https://github.com/AjuntamentdeBarcelona/decidim/issues/228